input/fsh/conceptmaps/CM_MaritalStatus.fsh

```diff
@@ -11,6 +12,7 @@ Usage: #definition
 * insert MapConcept( #M,  "Married",#M, "Married")
 * insert MapConcept( #S,  "Never Married",#S, "Never Married")
 * insert MapConcept( #W,  "Widowed",#W, "Widowed")
-//* insert MapConcept( #T,  "Domestic Partnership",#T, "Domestic Partnership") -- not yet supported by NCHS
 * insert AddGroup($IJE,$v3-NullFlavor)
 * insert MapConcept( #U,  "Not Classifiable",#UNK, "unknown")
+* insert MapConcept( #U,  "Not Classifiable",#T, "Domestic Partnership") //-- not yet supported by NCHS
```

Comment:
Are you allowed to have a concept with > 1 mapping in a ConceptMap?

Reply:
I don't see examples of it, it may be because `mappings.cs` in the library have IJEToFHIR and FHIRToIJE mappings. The IJEToFHIR mapping would just have one less mapping (no U -> T mapping because NCHS does not support it). Not sure how the mappings are generated from the IG though.

Reply:
Pete confirmed in Slack it is doable on the software side. The publisher does not complain and the wording on the [mapping page](https://build.fhir.org/conceptmap.html#notes) suggests it is possible, but should be clear. I added some documentation.